tools/generator/src/DTO/XcodeScheme.swift

```diff
@@ -20,11 +28,112 @@ struct XcodeScheme: Equatable, Decodable {
     }
 }
 
+extension XcodeScheme {
+    /// Create a new scheme applying any default actions based upon the current scheme.
+    var withDefaults: XcodeScheme {
+        get throws {
+            var buildTargets = [BazelLabel: XcodeScheme.BuildTarget]()
+
+            func enableBuildForValue(
+                _ label: BazelLabel,
+                _ keyPath: WritableKeyPath<XcodeScheme.BuildFor, XcodeScheme.BuildFor.Value>
+            ) throws {
+                do {
+                    try buildTargets[label, default: .init(label: label, buildFor: .init())]
+                        .buildFor[keyPath: keyPath]
+                        .merge(with: .enabled)
+                } catch XcodeScheme.BuildFor.Value.ValueError.incompatibleMerge {
+                    throw UsageError(message: """
+The buildFor value, "\(keyPath.stringValue)", for "\(label)" in the "\(name)" Xcode scheme was \
```

Comment:
We are exposing the value as `build_for` in Starlark, right?
```suggestion
The `build_for` value, "\(keyPath.stringValue)", for "\(label)" in the "\(name)" Xcode scheme was \
```

Reply:
Done